openmetadata-service/src/main/java/org/openmetadata/service/resources/dqtests/TestCaseResource.java

```diff
@@ -465,10 +465,11 @@ public Response addTestCaseResult(
           String fqn,
       @Valid TestCaseResult testCaseResult)
       throws IOException {
-    ResourceContextInterface resourceContext = TestCaseResourceContext.builder().name(fqn).build();
+    String decodedFQN = java.net.URLDecoder.decode(fqn, StandardCharsets.UTF_8);
```

Comment:
its up to the client to pass a proper FQN. Lets not add this on backend

Reply:
makes sense
cc: @open-metadata/ui